lib/router.js

```diff
@@ -20,15 +20,15 @@ exports.getRouter = function () {
 exports.setupAdminRoutes = function () {
 
     // map forgotten password
-    router.get(linz.get('admin forgot password path'), helpers.linz(), routes.forgottenPassword.get);
-    router.post(linz.get('admin forgot password path'), helpers.linz(), middleware.forgottenPassword.post, routes.forgottenPassword.post);
+    router.get(linz.get('admin forgot password path'), routes.forgottenPassword.get);
+    router.post(linz.get('admin forgot password path'), middleware.forgottenPassword.post, routes.forgottenPassword.post);
 
     // map password reset
-    router.get(linz.get('admin password reset path') + '/:id/:hash', helpers.linz(), middleware.passwordReset.get, routes.passwordReset.get);
-    router.post(linz.get('admin password reset path'), helpers.linz(), middleware.passwordReset.post, routes.passwordReset.post);
+    router.get(linz.get('admin password reset path') + '/:id/:hash', middleware.passwordReset.get, routes.passwordReset.get);
+    router.post(linz.get('admin password reset path'), middleware.passwordReset.post, routes.passwordReset.post);
 
-	router.get('*', helpers.linz(), middleware.adminEnsureAuthenticated());
-	router.post('*', helpers.linz(), middleware.adminEnsureAuthenticated());
+	router.get('*', middleware.linz, middleware.navigation, middleware.adminEnsureAuthenticated());
```

Comment:
Was just reading this, not sure if possible, but it seems to make sense that the current order of `middleware.navigation` and then `middleware.adminEnsureAuthenticated()` would be swapped to `middleware.adminEnsureAuthenticated(),middleware.navigation`. Why generate the navigation unless they're authenticated?